testcafe/tests/photos_crud.js

```diff
@@ -0,0 +1,255 @@
+import { Selector, Role } from "testcafe"; //import testcafe function
+import { regularUser } from "./helpers/roles"; //import roles for login
+import config from "../config"; //import url & psswd config
+//import { generateRandomInteger } from "./helpers/utils.js";
+const _ = require("lodash");
+
+import Page from "./pages/photos-model";
+
+const page = new Page();
+
+fixture`PHOTOS - CRUD`.page`${config.photosUrl}`.beforeEach(async t => {
+  await t.useRole(regularUser);
+  await t.resizeWindow(1280, 1024); // No upload button on mobile view, beware of the size!
+
+  t.ctx.allPhotosStartCount = await page.allPhotos.count; //Pics count at test start
+  console.log("beforeEach > allPhotosStartCount " + t.ctx.allPhotosStartCount);
+});
+
+test("Uploading 1 pic from Photos view", async t => {
+  //new pic shows up
+
+  await t
+    .setFilesToUpload(page.btnUpload, ["../data/IMG0.jpg"])
+    .expect(page.divUpload.visible)
+    .ok()
+    .expect(page.modalUpload.exists)
+    .ok({ timeout: 50000 })
+    .expect(page.divUpload.child("h4").innerText)
+    .contains("Uploaded 1 out of 1 successfully");
+  await t.takeScreenshot();
+
+  const allPhotosEndCount = await page.allPhotos.count; //Pics count at the end
+  console.log("allPhotosEndCount " + allPhotosEndCount);
+  await t.expect(allPhotosEndCount).eql(t.ctx.allPhotosStartCount + 1);
+});
+
+test("Uploading 3 pcis from Photos view", async t => {
+  //new pics show up
+
+  await t
+    .setFilesToUpload(page.btnUpload, [
+      "../data/IMG-JPG.jpg",
+      "../data/IMG-PNG.png",
+      "../data/IMG-GIF.gif"
+    ])
+    .expect(page.divUpload.visible)
+    .ok()
+    .expect(page.modalUpload.exists)
+    .ok({ timeout: 50000 })
+    .expect(page.divUpload.child("h4").innerText)
+    .contains("Uploaded 3 out of 3 successfully");
```

Comment:
I think this is something we do not want neither. We can have minor change on our strings. 

What do you think about using our translation system here to check the string ?

Reply:
I think this need to change too! 
Actually the part I really need is the "3" in "3 ou of 3 succesfully") because even if the upload fail, we still have the same message but with "0 out of 3"
So maybe a regular expression checking we have "3" twice in the string will be enough